stun_server/region/ecs.tf

```diff
@@ -6,26 +6,27 @@ resource "aws_ecs_service" "stun-server" {
   deployment_minimum_healthy_percent = 100
   deployment_maximum_percent         = 200
   launch_type                        = "FARGATE"
-
-  # Required to fetch the public IP address of the ECS service
-  enable_ecs_managed_tags = true
-  wait_for_steady_state   = true
+  depends_on                         = [aws_lb_listener.tcp, aws_lb_listener.udp]
 
   network_configuration {
-    assign_public_ip = true
+    assign_public_ip = false
     security_groups  = [aws_security_group.stun_sg.id]
-    subnets          = local.infrastructure_region_outputs.public_subnets
+    subnets          = local.infrastructure_region_outputs.private_subnets
   }
 
-  tags = {
-    region = data.aws_region.current.name
+  load_balancer {
+    container_name   = local.service_name
+    container_port   = "3478"
+    target_group_arn = aws_lb_target_group.tcp.arn
   }
-}
 
-data "aws_network_interface" "stun_server_interface" {
-  filter {
-    name   = "tag:aws:ecs:serviceName"
-    values = [aws_ecs_service.stun-server.name]
+  load_balancer {
+    container_name   = local.service_name
+    container_port   = "3478"
+    target_group_arn = aws_lb_target_group.udp.arn
+  }
```

Comment:
Have to check how/if this is going to work because, as far as I understand, ECS supports UDP, but you cannot specify the protocol for the load balancer.

Reply:
Not relevant anymore.